app/graphql/types/flow_type_setting_type.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Types
+  class FlowTypeSettingType < Types::BaseObject
+    description 'Represents a flow type setting'
+
+    authorize :read_flow_type_setting
+
+    field :data_type, Types::DataTypeType, null: true, description: 'Data type of the flow type setting'
+    field :descriptions, Types::TranslationType.connection_type, null: true,
```

Comment:
I think for the translation we should be ok with directly using an array. I don't think that we will need pagination for the translations of a single object. Also, I think those can be not-null as the list will be empty if no translations are present. (This suggestion also applies to the other fields returning translations)
```suggestion
    field :descriptions, [Types::TranslationType], null: false,
```